### ISSUE TITLE
Legal Definition Conflict in License

### ISSUE THREAD

**Issue by LokiFawkes** [WinampDesktop/winamp#134]
The License is clearly proprietary, forbidding distribution and public modification, only allowing users to read the source code and to privately run their modified versions. However, it describes itself as copyleft.

This is a potential conflict in legal definition, as the definition of copyleft as accepted by society at large forbids this type of license. This type of conflict has **undefined behavior** in copyright law. A court could, for example, rule that your license is miselading and unenfoceable, ranging from a need to edit the License once again to make it enforceable, to invalidating the License entirely and releasing the code to the hands of the public.

In addition, if anyone forks Winamp as part of the process of creating and testing modifications to be staged for a pull request, they would run afoul of the license for the same reason as above - It forbids distribution, and forking on Github is distribution, but also a necessary part of contribution.

Suggested fix: Change to a copyleft license as promised by the preamble of the License, or move Winamp's source code to a more restrictive platform, such as an internally controlled gitlab server.

For sources, feel free to look up Copyleft on Wikipedia or Law Insider or contact a copyright attorney.

**Comment by ninjaraven** [comment]
too long, didn't read
![bashame-meme](https://github.com/user-attachments/assets/626e9675-51e8-4dbb-8f53-acf0770c42a1)

**Comment by LokiFawkes** [comment]
TLDR: IANAL but the License is almost definitely illegal

**Comment by ninjaraven** [comment]
thanks, I get it now
![bashame-bashame-meme](https://github.com/user-attachments/assets/72c651da-2064-401e-a209-80ca8ba439c8)

**Comment by ivelasquez-rl** [comment]
amazing how this is so important but completely forgotten

**Comment by Poikilos** [comment]
IANAL but here are some widely accepted definitions that basically all seem to be the same, but differ from the WCL:
- https://www.gnu.org/licenses/copyleft.en.html
- https://www.lsd.law/define/copyleft
- https://www.britannica.com/topic/copyleft
- and various oxford publications: https://www.oxfordreference.com/search?q=copyleft&searchBtn=Search&isQuickSearch=true

**Comment by dfskoll** [comment]
Yeah.  This license is bulshit.  It's not by any means a free or open-source license.  The license is essentially:  "Do the work for us.  We'll take all of your work and give you nothing in return, and oh by the way... we now own the copyright to your work."

**Comment by LokiFawkes** [comment]
This should be reopened. The repo remains illegal per its license and status as a public repo.
